common/connector.go

```diff
@@ -124,3 +128,100 @@ type LightningConnector interface {
 	// connector.
 	FundsAvailable() (decimal.Decimal, error)
 }
+
+// PaymentStore is used to store payments for external services (except
+// exchange, which is explicitly notified) for give them ability to
```

Comment:
Payment store as an entity should know nothing about system being notified about payment, so please put comment which describe it in more general terms.

Reply:
fixed